src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/RazorSemanticTokenInfoServiceTest.cs

```diff
@@ -57,11 +53,14 @@ public async Task GetSemanticTokens_CSharpBlock_HTML()
                 """;
 
         var razorRange = GetRange(documentText);
-        var csharpTokens = await GetCSharpSemanticTokensResponseAsync(documentText, razorRange, isRazorFile: false);
-        await AssertSemanticTokensAsync(documentText, isRazorFile: false, razorRange, csharpTokens: csharpTokens);
+        var (csResponse, perRangeTokens) = await GetCSharpSemanticTokensResponseAsync(documentText, razorRange, isRazorFile: false);
+        await AssertSemanticTokensAsync(documentText, isRazorFile: false, razorRange, csResponse, perRangeTokens);
+        Assert.NotNull(csResponse.Tokens);
+        Assert.NotEmpty(csResponse.Tokens);
     }
 
-    [Fact]
+    [Fact(Skip = "For some reason [12, 38, 15, 18, 0] has discrepancy with original tokens on the third index, not contained in [[12, 38, 16, 18, 0], [12, 54, 1, 54, 0], ...]. Test is skipped to debug further.")]
```

Comment:
Does this actually fail? Or just have different results between feature  flag and non? What does this discrepancy look like in the test baseline files? Should be easy to tell what is going wrong because they include the actual test being classified.

Reply:
Discussed this offline with @davidwengier and we found that the difference in behavior here is related to the difference we see in the two baseline files (when comparing the result of two feature flags together).

We agreed to remove this test and that simplifies the test file quite a lot as well.